packages/card/src/Card.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const containerStyle = css`
+  background-color: white;
+  border: 1px solid ${uiColors.gray.light2};
+  box-shadow: 0px 4px 10px -4px ${uiColors.gray.light1};
+  transition: border 300ms ease-in-out, box-shadow 300ms ease-in-out;
+
+  &:hover {
+    border: 1px solid ${uiColors.gray.light2};
+    box-shadow: 0px 3px 6px -2px ${uiColors.gray.base};
+  }
+`;
+
+interface CardProps {
+  children: React.ReactNode;
+  as?: React.ElementType<any>;
```

Comment:
I think TS has all the available DOM elements that we can include as the generic value.

Reply:
But what if someone wants to use as a <Link> element or something like that

Reply:
Hi @hswolff! Want to merge this in, but first wanted to make sure you were okay with leaving as-is. If you disagree, I'll update the type!

Reply:
Couldn't we have TS support both? 

Also what does it look like to have a Link as a card? That seems very odd.